homeworks/rostyslav.havuka_RostyslavHavuka/11-dependency-injection/vehicle/src/app/app.component.html

```diff
@@ -0,0 +1,11 @@
+<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
+<mat-spinner *ngIf="areUsersLoading" style="margin: auto; margin-top: 10%;"></mat-spinner>
+
+<ul *ngIf="!areUsersLoading">
+  <li *ngFor="let car of cars" class="mat-subheading-2">
+   <a class="mat-raised-button" href="https://www.google.com/search?q={{ car.vehicle }}" http>Added from <a href="https://www.google.com/search?q={{ car.vehicle }}">{{ car | protocolCar }}</a></a><br>
```

Comment:
Link nested in link? 😲
Change it only to the one.

Also, encode your link (you can create a pipe for it)